src/__tests__/randomThings.test.js

```diff
@@ -0,0 +1,71 @@
+import * as rt from '../app/randomThings'
+
+describe('random number generation...', () => {
+  it('returns random int within requested (inclusive) limits', () => {
+    const randomInteger = rt.randomInteger(0, 1)
+
+    expect(Number.isInteger(randomInteger)).toBe(true)
+
+    expect(randomInteger).toBeGreaterThanOrEqual(0)
+    expect(randomInteger).toBeLessThanOrEqual(1)
+
+    expect(rt.randomInteger(0, 0)).toEqual(0)
+  })
+
+  it('errors when requesting random int with invalid params', () => {
+    expect(() => { rt.randomInteger(0) }).toThrow()
+    expect(() => { rt.randomInteger(1, 0) }).toThrow()
+    expect(() => { rt.randomInteger(':)', 0) }).toThrow()
+  })
+
+  it('returns sorted array of two random ints within req (incl) limits', () => {
+    const randomRange = rt.randomRange(0, 9)
+
+    expect(randomRange instanceof Array).toBe(true)
+
```

Comment:
just curious: why the new lines?

Reply:
I like to isolate variable declaration, then logically (to me, anyway) by related functionality. Using that function as an example:

```
it('returns random int within requested (inclusive) limits', () => {
  const randomInteger = rt.randomInteger(0, 1)

  // type check
  expect(Number.isInteger(randomInteger)).toBe(true)

  // within limits check
  expect(randomInteger).toBeGreaterThanOrEqual(0)
  expect(randomInteger).toBeLessThanOrEqual(1)

  // exact result check
  expect(rt.randomInteger(0, 0)).toEqual(0)
})

```